src/nv_ingest/modules/sinks/vdb_task_sink.py

```diff
@@ -34,6 +39,61 @@
 
 VDBTaskSinkLoaderFactory = ModuleLoaderFactory(MODULE_NAME, MODULE_NAMESPACE, VdbTaskSinkSchema)
 
+_DEFAULT_ENDPOINT = os.environ.get("MINIO_INTERNAL_ADDRESS", "minio:9000")
```

Comment:
I think we solved this, as leaving it in place, because Milvus only works with minio. And milvus is the only Vector Database that supports GPU indexes right now.

Reply:
That's fine, the issue is really that I want to avoid gathering environment variables in stages vs having them passed in to the stage as configuration parameters that can be validated.